UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -0,0 +1,182 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using TileManagement;
+using UnityEngine;
+using System.Linq;
+
+public static class MapSaver
+{
+	public readonly static char Matrix4x4Char = '#' ;
```

Comment:
now that they are in the same class and this class does serialization and deserialization, I don't expect these to be needed elsewhere, right? If that's the case, make then private, otherwise say and I resolve the conversation.

Reply:
it might be a separate class called map loader for the loading